packages/framework/block-std/src/gfx/surface/decorators/required.ts

```diff
@@ -0,0 +1,58 @@
+import type { Constructor } from '@blocksuite/global/utils';
```

Comment:
This should be in the `block-std/src/view/decorators` folder

Reply:
Good idea! By the way, there was a lot of use of `!` in LitComponent now, I'm not sure if this `@requiredProperties` refactoring is necessary.